generators/languages/generator.ts

```diff
@@ -212,6 +212,54 @@ export default class LanguagesGenerator extends BaseApplicationGenerator<
           });
         };
       },
+      updateLanguages({ source }) {
+        source.addLanguages = ({ languagesDefinition }: { languagesDefinition: readonly Language[] }) => {
+          if (languagesDefinition && languagesDefinition.length > 0) {
+            (source as unknown as ClientSource).addLanguageWithDefinition?.({ languagesDefinition });
+          }
+        };
+      },
+      updateLanguageAngular({ source, application }) {
+        const { enableTranslation, clientFrameworkAngular, skipClient } = application;
+        if ((skipClient === undefined || !skipClient) && enableTranslation && clientFrameworkAngular) {
+          source.updateLanguagesClient = (gen, { application, control }) => {
+            const app = application as AngularApplication<AngularEntity>;
+            updateLanguagesInAngularTask.call(gen, { application: app, control });
+          };
+        }
+      },
+      updateLanguageReact({ source, application }) {
+        const { enableTranslation, clientFrameworkReact, skipClient } = application;
+        if ((skipClient === undefined || !skipClient) && enableTranslation && clientFrameworkReact) {
+          source.updateLanguagesClient = (gen, { application, control }) => {
+            const app = application as ClientApplication;
+            updateLanguagesInReactTask.call(gen, { application: app, control });
+          };
+        }
+      },
+      updateLanguageVue({ source, application }) {
+        const { enableTranslation, clientFrameworkVue, skipClient } = application;
+        if ((skipClient === undefined || !skipClient) && enableTranslation && clientFrameworkVue) {
+          source.updateLanguagesClient = (gen, { application, control }) => {
+            const app = application as ClientApplication;
+            updateLanguagesInVueTask.call(gen, { application: app, control });
+          };
+        }
+      },
```

Comment:
This may be moved to (angular/react/vue):bootstrap generators.

Reply:
I would like to, but:
1) there's no bootstrap gen there yet (and I'm not confident of the linkage if I do it myself)
2) While full generation would work, Standalone (cli) language addition won't really do (tried, didn't succeed)